### ISSUE TITLE
Support multi-language

### ISSUE THREAD

**Issue by Angelk90** [waseefakhtar/dose-android#120]
Hi @waseefakhtar , interesting project, does it support the possibility of having multiple languages?

**Comment by waseefakhtar** [comment]
Yes of course! The idea was to find the common languages among users and localize accordingly but everyone is free to contribute and translate into their own languages. :)
